components/elm/src/biogeochem/DUSTMod.F90

```diff
@@ -238,34 +237,47 @@ subroutine DustEmission (bounds, &
     real(r8) :: sumwt(bounds%begl:bounds%endl) ! sum of weights
     logical  :: found                          ! temporary for error check
     integer  :: index
-    !
+    !    
     ! constants
     !
-    real(r8), parameter :: cst_slt = 2.61_r8           ! [frc] Saltation constant
+    !real(r8), parameter :: cst_slt = 2.61_r8           ! [frc] Saltation constant, variable no longer needed -YF
     real(r8), parameter :: flx_mss_fdg_fct = 5.0e-4_r8 ! [frc] Empir. mass flx tuning eflx_lh_vegt
     real(r8), parameter :: vai_mbl_thr = 0.3_r8        ! [m2 m-2] VAI threshold quenching dust mobilization
-    !------------------------------------------------------------------------
+    real(r8), parameter :: Cd0 = 4.4e-5_r8             ! [dimless] proportionality constant 
+                                                       ! in calculation of dust emission coefficient -YF
+    real(r8), parameter :: Ca = 2.7_r8                 ! [dimless] proportionality constant in scaling of dust
+                                                       ! emission exponent -YF
+    real(r8), parameter :: Ce = 2.0_r8                 ! [dimless] proportionality constant scaling 
+                                                       ! exponential dependence of dust emission coefficient 
+                                                       ! on standardized soil threshold friction speed -YF
+    real(r8), parameter :: C_tune = 0.05_r8            ! [dimless] global tuning constant for vertical dust flux;
+                                                       ! set to produce ~same global dust flux in
+                                                       ! control sim (I_2000) as old parameterization -YF
+    real(r8), parameter :: wnd_frc_thr_slt_std_min = 0.16_r8 ! [m/s] minimum standardized soil threshold friction speed -YF
+    real(r8), parameter :: forc_rho_std = 1.2250_r8    ! [kg/m3] density of air at standard pressure (101325) and temperature (293 K) -YF
```

Comment:
I like the way you have units as well as an explanation of each of these variables!